### PR TITLE
fix: date picker last date

### LINKED ISSUE #983: DatePicker's highlighted date is set to current date even after user selects a different date.
**Summary:**

DatePicker's highlighted date is set to current date even after user selects a different date.

**Steps to reproduce:**
Goto->Accounts->Loan fragment
Choose floating action button to apply for loan
Set expected disbursement date to future value and open datepicker dialog again.

**Expected behaviour:**

DatePickers highlighted date should be the date selected by user previously

**Observed behaviour:**

DatePickers highlighted date is current date always

**Device and Android version:**

Motorola Moto G4 (Android 7.0,API 24)
**Screenshots:**

ezgif com-video-to-gif

Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.

### PR DESCRIPTION
**Pull request by laxyapahuja**
Fixes #983

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by laxyapahuja** [comment]
@luckyman20 @elirehema @jddeep @miplodder can you please review this task? :)

**Review by zoman-anamika — APPROVED** [review]

File: app/src/main/java/org/mifos/mobile/models/templates/loans/LoanTemplate.kt [hunk 1]
```diff
@@ -13,6 +13,7 @@ import org.mifos.mobile.models.accounts.loan.InterestRateFrequencyType
 import org.mifos.mobile.models.accounts.loan.InterestType
 import org.mifos.mobile.models.accounts.loan.RepaymentFrequencyType
 import org.mifos.mobile.models.accounts.loan.TermPeriodFrequencyType
+import java.io.Serializable
```
Comment (ghost) [inline app/src/main/java/org/mifos/mobile/models/templates/loans/LoanTemplate.kt]:
```suggestion
```
Unnecessary imports

**Comment by ghost** [comment]
@laxyapahuja please remove the unnecessary import

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@luckyman20 LGTM :+1:

**Pull request merged**